README.md

```diff
@@ -50,3 +50,25 @@ API Setup
 
 
 [AB2D Deploy](Deploy/README.md)
+
+## Installing and Using Pre-commit with Gitleaks
```

Comment:
Let's leave gitleaks out of this. Also this should be added to the Table of Contents above.

```suggestion
## Installing and using pre-commit
```